data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
```

Comment:
As far as I know, stuff that exists IRL should behave as close to it as possible in game, but it may be balanced by non-existant stuff if it's needed, which can behave in any sensible way. Well, at least that's how I understand it.